### PR TITLE
chore(windows): remove remaining unused Makefiles

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes: #12247

@keymanapp-test-bot skip

### CODE REVIEW

**Review by ermshiperete — APPROVED** [review]
rslgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.97-alpha](https://keyman.com/downloads/releases/alpha/18.0.97)

File: resources/build/win/environment.inc.sh [hunk 1]
```diff
@@ -23,8 +23,10 @@ source "$KEYMAN_ROOT/resources/build/win/environment_generated.inc.sh"
 WINDOWS_ROOT="$KEYMAN_ROOT/windows"
 WINDOWS_PROGRAM_APP="$WINDOWS_ROOT/bin/desktop"
 WINDOWS_PROGRAM_ENGINE="$WINDOWS_ROOT/bin/engine"
+WINDOWS_PROGRAM_SUPPORT="$WINDOWS_ROOT/bin/support"
 WINDOWS_DEBUGPATH_APP="$WINDOWS_ROOT/debug/desktop"
 WINDOWS_DEBUGPATH_ENGINE="$WINDOWS_ROOT/debug/engine"
+WINDOWS_DEBUGPATH_SUPPORT="$WINDOWS_ROOT/debug/suppor"
```
Comment (rc-swag) [inline resources/build/win/environment.inc.sh:29]: I know this has merged but there is a typo missing the "t" of support
"WINDOWS_DEBUGPATH_SUPPORT="$WINDOWS_ROOT/debug/support"

Comment (mcdurdin) [reply resources/build/win/environment.inc.sh:29]: See https://github.com/keymanapp/keyman/pull/12286

**Pull request merged**